lago/virt.py

```diff
@@ -219,6 +221,82 @@ def export_vms(self, vms_names, standalone, dst_dir, compress):
             for _vm in vms:
                 _vm.export_disks(standalone, dst_dir, compress)
 
+        self.generate_init(dst_dir, out_format)
+
+    def generate_init(self, dst_dir, out_format, filters=None):
+        """
+        Generate an init file which represents this env and can
+        be used with the images created by self.export_vms
+
+        Args:
+            dst_dir (str): path to the new init file
+            out_format (plugins.output.OutFormatPlugin):
+                formatter for the output
+            filters (list): list of paths to keys that should be removed from
+                the init file
+        Returns:
+            None
+        """
+        with LogTask('Exporting init file to: {}'.format(dst_dir)):
+            if not filters:
+                filters = [
+                    'domains/*/disks/*/metadata',
+                    'domains/*/metadata/deploy-scripts', 'domains/*/snapshots',
+                    'domains/*/name', 'nets/*/mapping'
+                ]
+            spec = self.get_env_spec(filters)
+
+            for _, domain in spec['domains'].viewitems():
+                for disk in domain['disks']:
+                    if disk['type'] == 'template':
+                        disk['template_type'] = 'qcow2'
+                    elif disk['type'] == 'empty':
+                        disk['type'] = 'file'
+                        disk['make_a_copy'] = 'True'
+
+                    # Insert the relative path to the exported images
+                    disk['path'] = os.path.join(
+                        '$LAGO_INITFILE_PATH', os.path.basename(disk['path'])
+                    )
+
+            with open(os.path.join(dst_dir, 'LagoInitFile'), 'wt') as f:
```

Comment:
The dumping filename should be configuable, moreover it should drop it to stdout by default(like ansible_hosts cmd), so you could do something like:
```
lago dump-init >> my-init-el73-2011
```

Not sure if its an issue but if the file already exists(LagoInitFile) it will override it.